### DISCUSSION TITLE
how to add floating label in select component ?

### DISCUSSION THREAD

**Discussion by akshatagawas06** [spartan-ng/spartan#170] [Q&A]
want to add floating label in HlmSelectTriggerComponent , is that possible ? how to implement that

**Comment by goetzrobin** [comment]
Can you describe a little more what you mean when saying you want a floating label. Select has a brain primitive that gives you complete freedom over the styling. In theory, you should be able to listen to the focus state of the trigger and then position/animate the label with CSS to fit your design requirements.
